scalalib/src/mill/scalalib/JavaModule.scala

```diff
@@ -191,6 +200,23 @@ trait JavaModule
       )
   }
 
+  /**
+   * BOM dependencies that are not meant to be overridden or changed by users.
+   *
+   * This is mainly used to add BOM dependencies of the main module to its test
+   * modules, while ensuring test dependencies of the BOM are taken into account too
+   * in the test module.
+   */
+  def extraBomIvyDeps: Task[Agg[BomDependency]] = Task.Anon { Agg.empty[BomDependency] }
+
+  /**
+   * Dependency management entries that are not meant to be overridden or changed by users.
+   *
+   * This is mainly used to add dependency management entries of the main module to its test
+   * modules.
+   */
+  def extraDepManagement: Task[Agg[Dep]] = Task.Anon { Agg.empty[Dep] }
```

Comment:
Do these really need to be `Task.Anon`s? Not sure about `BomDependency`, but at least `Agg[Dep]` is serializable and can be a normal `Task` right?

Reply:
These can be made normal `Task`s, yes